MariaDB/server
# MDEV-28145: columnstore still enabled in i386 builds

- [x] *The Jira issue number for this PR is: MDEV-28145*


## Description

This cannot work due to undefined int128 types in 32 builds.

As such we take from the RocksDB and explicitly disable when the
feature isn't there.

## How can this PR be tested?

see if bb fails on i386 builds.

Also pay attention to non-amd64/arm64 builds to see if they previously built and now don't

## Basing the PR against the correct MariaDB version
- [ X ] *This is a bug fix and the PR is based against the earliest branch in which the bug can be reproduced*


## Backward compatibility

Check if ppc64le, or other non-amd64/arm64 builds fail.
